### ISSUE TITLE
Wrong Directories in Basic Usage Tutorial

### ISSUE THREAD

**Issue by kinifwyne** [nifty-site-manager/nifty-site-manager.github.io#2]
I just completed the [Basic Usage Tutorial](https://nifty-site-manager.github.io/resources/tutorials/basic_usage.html), and I got confused on step 10 because there was no `.siteinfo/pages.list`. I eventually found the file under `.nsm/tracking.info`. For some reason, directories and files in this tutorial are misnamed. In the file I downloaded, I found:
- `.nsm/` instead of `.siteinfo/`
    - `.nsm/tracking.info` instead of `.siteinfo/pages.info`
- `output/` instead of `site/`

I did notice that the output directory was set to `output/` in  `.nsm/nift.config`, but nowhere in the tutorial is this mentioned.

**Comment by kinifwyne** [comment]
When I did step 34, `output/about.html` and `.nsm/output/about.info` were removed instead of `site/index.html` and `.siteinfo/site/index.info`.

**Comment by n-ham** [comment]
Oops, I must have forgotten to update those when I changed the naming structure last year, my apologies. I have hopefully updated the tutorials now. 

The tutorials are a bit basic sorry, the documentation however is reasonably thorough so I recommend checking that out once you have a basic idea about how Nift works. The [functions](https://nift.dev/docs/fns.html) docs are the most useful, however the docs pages for Commands, Types, f++, n++, Lua(JIT), ExprTK, Interactive REPL and Scripts are especially worth checking out.

If you spot any further issues or have any questions let me know, I seem to be able to allow discussions on the main nsm repo now so I have enabled those as well. 

I had been taking a bit of a break from Nift in the second half of 2020, however am keen to do some more work on it in 2021.
